src/Microsoft.Graph/Requests/Generated/AdministrativeUnitRequest.cs

```diff
@@ -117,9 +117,23 @@ public System.Threading.Tasks.Task<AdministrativeUnit> UpdateAsync(Administrativ
         /// </summary>
         /// <param name="administrativeUnitToUpdate">The AdministrativeUnit to update.</param>
         /// <param name="cancellationToken">The <see cref="CancellationToken"/> for the request.</param>
+		/// <exception cref="ClientException">Thrown when an object returned in a response is used for updating an object in Microsoft Graph.</exception>
```

Comment:
Can we fix this indentation issue in our code gen template.

Reply:
Thanks @peombwa, I've corrected this; I've now used space in place of tabs.